metatron/src/main/java/uk/gov/ida/eidas/metatron/resources/MetatronResource.java

```diff
@@ -1,35 +1,50 @@
 package uk.gov.ida.eidas.metatron.resources;
 
-import uk.gov.ida.eidas.metatron.core.dto.EidasConfig;
-import uk.gov.ida.eidas.metatron.core.dto.EidasCountryConfig;
+import net.shibboleth.utilities.java.support.resolver.CriteriaSet;
+import net.shibboleth.utilities.java.support.resolver.ResolverException;
+import org.opensaml.core.criterion.EntityIdCriterion;
+import org.opensaml.saml.saml2.metadata.EntityDescriptor;
+import uk.gov.ida.eidas.metatron.domain.EidasConfig;
+import uk.gov.ida.eidas.metatron.domain.EidasCountryConfig;
+import uk.gov.ida.eidas.metatron.domain.MetadataResolverService;
+import uk.gov.ida.eidas.metatron.health.MetadataHealth;
 
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
 import java.security.KeyStore;
 import java.security.KeyStoreException;
 import java.util.Iterator;
+import java.util.List;
 
 @Path("/")
 public class MetatronResource {
 
     EidasConfig config;
+    MetadataResolverService metadataResolverService;
 
-    public MetatronResource(EidasConfig config) {
+    public MetatronResource(EidasConfig config, MetadataResolverService metadataResolverService) {
         this.config = config;
+        this.metadataResolverService = metadataResolverService;
     }
 
     @GET
     @Path("/config")
     public String config() {
         return config.getCountries().stream()
                 .map(EidasCountryConfig::toString)
-                .reduce("", (in, config) -> in + config) ;
+                .reduce("", (in, config) -> in + config);
     }
 
     @GET
-    @Path("/truststore")
-    public String truststore() throws KeyStoreException {
-        KeyStore truststore = config.getKeyStore();
+    @Path("/metadata-truststore/{country}")
```

Comment:
This route is for testing right? We'll probably remove it before Metatron is released to the wild?